buildahimage/README.md

```diff
@@ -0,0 +1,42 @@
+![buildah logo](https://cdn.rawgit.com/containers/buildah/master/logos/buildah-logo_large.png)
+
+# buildahimage
+
+## Overview
+
+This directory contains the Dockerfiles necessary to create the three buildahimage container
+images that are housed on quay.io under the buildah account.  All three repositories where
+the images live are public and can be pulled without credentials.  These container images are secured and the
+resulting containers can run safely with privileges within the container.  The container images are built
+using the latest Fedora and then Buildah is installed into them:
+
+  * quay.io/buildah/buildahimage - This image is built using the latest stable version of Buildah in a Fedora based container.  Built with Dockerfile.stable.
+  * quay.io/buildah/buildahimagegit - This image is built using the latest code found in this GitHub repository.  When someone creates a commit and pushes it, the image is created.  Due to that the image changes frequently and is not guaranteed to be stable.  Built with Dockerfile.git.
+  * quay.io/buildah/buildahimagetesting - This image is built using the latest version of Buildah that is or was in updates testing for Fedora.  At times this may be the same as the stable image.  This container image will primarily be used by the development teams for verification testing when a new package is created.  Built with Dockerfile.testing.
+
+## Sample Usage
+
+Although not required, it is suggested that [Podman](https://github.com/containers/libpod) be used with these container images.
+
+```
+podman pull docker://quay.io/buildah/buildahimage:latest
+
+podman run buildahimage buildah version
+
+mkdir /var/lib/mycontainer
```

Comment:
Can we add a comment why we need this volume?

Reply:
It also implies we're running as root which seems like a trap users can step into if they try rootless.

Reply:
I've added a note about the volume.
It's a good implication for root.  At the moment rootless is a bit sketchy.  Will come back later on and rework the examples once that's a bit steadier.